src/main/java/network/brightspots/rcv/DominionCvrReader.java

```diff
@@ -230,30 +232,48 @@ private void validateNamesAreInContest(List<CastVoteRecord> castVoteRecords)
   private void gatherCvrsForContest(List<CastVoteRecord> castVoteRecords, String contestIdToLoad) {
     try {
       Path singleCvrPath = Paths.get(cvrPath, CVR_EXPORT);
-      Path firstCvrPath = Paths.get(cvrPath, String.format(CVR_EXPORT_PATTERN, 0));
+
       if (singleCvrPath.toFile().exists()) {
         HashMap json = JsonParser.readFromFile(singleCvrPath.toString(), HashMap.class);
         parseCvrFile(json, castVoteRecords, contestIdToLoad);
-      } else if (firstCvrPath.toFile().exists()) {
+      } else {
+        // We are expecting multiple CvrExport_N.json files
+        Pattern pattern = Pattern.compile("^CvrExport_\\d+\\.json$");
+        HashSet<File> matchedCvrFiles = new HashSet<>();
+
+        // Use regex to find all CvrExport_N.json files
+        File[] files = Paths.get(cvrPath).toFile().listFiles();
+        if (files != null) {
+          for (File file : files) {
+            Matcher matcher = pattern.matcher(file.getName());
+            if (matcher.matches()) {
+              matchedCvrFiles.add(file);
+            }
+          }
+        }
+
+        if (matchedCvrFiles.isEmpty()) {
+          String errorMessage = "Error parsing Dominion cast vote records:"
+                  + " CvrExport.json file(s) not located";
+          throw new FileNotFoundException(errorMessage);
+        }
+
         int recordsParsed = 0;
+        int filesParsed = 0;
         int recordsParsedAtLastLog = 0;
-        int cvrSequence = 0;
-        Path cvrFilePath = Paths.get(cvrPath, String.format(CVR_EXPORT_PATTERN, cvrSequence));
-        while (cvrFilePath.toFile().exists()) {
-          HashMap json = JsonParser.readFromFile(cvrFilePath.toString(), HashMap.class);
+
+        for (File file : matchedCvrFiles) {
```

Comment:
Could order of files read matter here? I fear it might with some of the randomizers, but not totally sure if any randomizer works at the CVR level.

For safety, I might sort the `matchedCvrFiles` just in case it does matter, now or later, so we generate the records in a deterministic order.